static/js/settings_invites.js

```diff
@@ -52,7 +52,7 @@ function populate_invites(invites_data) {
         modifier: function (item) {
             item.invited_absolute_time = timerender.absolute_time(item.invited * 1000);
             item.is_admin = page_params.is_admin;
-            item.disable_buttons = item.invited_as === 4 && !page_params.is_owner;
+            item.disable_buttons = item.invited_as === 100 && !page_params.is_owner;
```

Comment:
Please access this programmatically; e.g. via `settings_config.user_roles_values.owner.code`.